src/event_imp.rs

```diff
@@ -200,14 +227,20 @@ impl Ready {
         self.contains(Ready::writable())
     }
 
+    #[deprecated(since = "0.6.5", note = "use unix::ReadyExt instead")]
```

Comment:
This may be painful for a bit b/c the inherent method will trump the trait by default, so you'd have to call this like `ReadyExt::is_error(&ready)`

(just FYI)